Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3302,13 +3302,12 @@ def fetch_threats(client: Client, args):
     incidents_threats = []
     current_fetch = args.get('current_fetch')
     incident_statuses = args.get('fetch_threat_incident_statuses')
-    resolved = 'true' if incident_statuses and 'RESOLVED' in incident_statuses else 'false'
 
     threats = client.get_threats_request(limit=args.get('fetch_limit'),
                                          created_after=args.get('last_fetch_date_string'),
                                          site_ids=args.get('fetch_site_ids'),
                                          incident_statuses=','.join(incident_statuses).lower() if incident_statuses else None,
-                                         resolved=resolved)
+                                         include_resolved_param=False)
```

Comment:
Can you please explain what is the reason for this change?

Reply:
Required behaviour would be, Content pack should fetch the selected incident statues of the threat provide in the config. If we provide the resolve parameter that going to override the incident_statuses in the get threats api call to server. But that is not a required behaviour this a bug we found recently in the content pack so we fixed it.